ev3dev/brickpi.py

```diff
@@ -53,6 +53,7 @@ class Leds(object):
     LED1 = ( blue_led1, )
     LED2 = ( blue_led2, )
 
+    BLACK = ( 0, )
```

Comment:
I though about this, but then we need to call BLUE -> ON instead. I don't mind adding ON/OFF aliases, since this way we have a 'portable' way to turn an led on/off regardless of the type (that is ON is going to be as ON as possible :))